inc/policydeployapplication.class.php

```diff
@@ -241,4 +241,10 @@ public function showValue(PluginFlyvemdmTask $task) {
       }
       return NOT_AVAILABLE;
    }
+
+   public static function getEnumSpecificStatus() {
+      return [
+         'waiting' => __('Waiting', 'flyvemdm'),
+      ];
+   }
```

Comment:
@rafaelje: this status is for appp / file deployment and removal. If you need a user action, you may send this status to the backend to tell you need to wait for something.

Reply:
Fine